Tests/configure_and_test_integration_instances.py

```diff
@@ -345,7 +345,7 @@ def get_changed_integrations(self, packs_not_to_install: set[str] | None = None)
     def concurrently_run_function_on_servers(self, function=None, pack_path=None, service_account=None) -> tuple[bool, list[Any]]:
         raise NotImplementedError
 
-    def install_packs(self, pack_ids: list | None = None, multithreading=True, production_bucket: bool = True) -> bool:
+    def install_packs(self, pack_ids: list | None = None, install_packs_in_batches=False, production_bucket: bool = True) -> bool:
```

Comment:
Hi,
Why this change is necessary?

Reply:
Because following my changes multithreading is not used, and the only difference is whether to install in batches or not.